### PR TITLE
Fixed several strings and lack of gettext usage.

### PR DESCRIPTION
**Pull request by onpon4**
Per #1021

### CODE REVIEW

File: src/land.c [hunk 1]
```diff
@@ -176,14 +176,19 @@ int can_swapEquipment( char* shipname )
...
    if (strcmp(shipname,player.p->name)==0) { /* Already onboard. */
       land_errDialogueBuild( _("You're already onboard the %s."), shipname );
       failure = 1;
    }
    if (pilot_cargoUsed(player.p) > (pilot_cargoFree(newship) + pilot_cargoUsed(newship))) { /* Current ship has too much cargo. */
-      land_errDialogueBuild( _("You have %d tons more cargo than the new ship can hold."),
-            pilot_cargoUsed(player.p) - pilot_cargoFree(newship), shipname );
+      diff = pilot_cargoUsed(player.p) - pilot_cargoFree(newship);
+      land_errDialogueBuild( ngettext(
+               "You have %d tonnes more cargo than the new ship can hold.",
```
Comment (bobbens) [inline src/land.c:188]: Shouldn't this line be tonne?

Comment (onpon4) [reply src/land.c:188]: Ah, yup, lemme fix that...

**Comment by onpon4** [comment]
There we go. Sorry, I was copy-pasting those and I must have forgot to change "tonnes" to "tonne" in that case.

**Comment by onpon4** [comment]
Just pushed a fix of the tutorial mission as well (some things that were less than ideal). Also shifted the osd down in Brushed a bit.

**Comment by bobbens** [comment]
Looks good, thanks!

**Pull request merged**